system_tests/utils.go

```diff
@@ -749,7 +749,7 @@ func publishAndConsumeMQTTMsg(hostname, nodePort, username, password string, ove
 		fmt.Printf("Attempt #%d to connect using MQTT\n", retry)
 		token = c.Connect()
 		// Waits for the network request to reach the destination and receive a response
-		Expect(token.WaitTimeout(3 * time.Second)).To(BeTrue())
+		token.WaitTimeout(3 * time.Second)
```

Comment:
Could this entire loop just be replaced with a Gomega `Eventually` Statement? - https://onsi.github.io/gomega/#eventually

Reply:
Sure it can 😄 4863a88